format/normalize.go

```diff
@@ -0,0 +1,25 @@
+package format
+
+import "math"
+
+func Normalize(vec []float64) []float64 {
```

Comment:
We can just put this next to the code that's using it in `server` without exporting it `normalize`